web/components/FcDrawerViewStudyReports.vue

```diff
@@ -29,13 +29,6 @@
               &#x2022; {{locationActive.description}}
             </span>
           </h2>
-          <template v-if="filterChipsStudyNoStudyTypes.length > 0">
```

Comment:
here we can drop the computed property `filterChipsStudyNoStudyTypes`, as well as the getter `filterChipsStudy`